docs/variants/protectli_fw6/building-manual.md

```diff
@@ -0,0 +1,35 @@
+# Building manual
```

Comment:
Build instructions for Protectli were unified in https://github.com/Dasharo/docs/pull/851. Don't re-add this file, modify https://github.com/Dasharo/docs/blob/master/docs/unified/protectli/building-manual.md instead.

Reply:
Changed in: d0e64d58a768bd1ac17b02f724b3524074553fa4